snippets/language-ruby.cson

```diff
@@ -15,6 +15,9 @@
   'case … end':
     'prefix': 'case'
     'body': 'case ${1:object}\nwhen ${2:condition}\n\t$0\nend'
+  'begin … rescue … end':
+    'prefix': 'bre'
```

Comment:
Glad to see it get added here though. I should have done so long ago instead of just adding it locally.